epicJWTforBackend.js

```diff
@@ -1,58 +1,60 @@
-// Add to code templates https://raw.githubusercontent.com/kjur/jsrsasign/master/jsrsasign-all-min.js
-// Tested with version 10.7.0 
-
-// IMPORTANT In the code template you added you must add this near the top because this library can be used with browsers.
-// var navigator = {}  // fake the fact we have no browser
-// var window = {}     //
-
-// On Epic dev portal, you will need to upload your public key, their documentation is clear 
-// on generating your required rsa private and public keys and retreive your ClientID
-
-// Transformer code:
-
-// time
-const iat = Math.floor(Date.now() / 1000);
-
-// Set up the JWT header and payload
-var header =
-    {
+/**
+ * Generates a JWT for Epic FHIR server authentication.
+ * Requires https://raw.githubusercontent.com/kjur/jsrsasign/master/jsrsasign-all-min.js as a codetemplate
+ * Tested with version 10.7.0
+ * IMPORTANT In the code template you added you must add this near the top because this library can be used with browsers.
+ * var navigator = window = {}  // fake the fact we have no browser
+ *
+ * On Epic dev portal, you will need to upload your public key, their documentation is clear 
+ * on generating your required rsa private and public keys and retreive your ClientID
+ *
+ * Using an http sender
```

Comment:
Can an HTTP connector be exported and included in this PR to show the settings?

Reply:
Can it?  Yes.   Do I want to write everything for the user?  No.